react/components/AutocompleteResults/styles.css

```diff
@@ -0,0 +1,27 @@
+.resultsList {
+  list-style-type: none;
+  max-height: 60vh;
+  overflow-y: auto;
+  top: 100%;
```

Comment:
I imagine that this handle is being used through `handles.resultsList`. You could use some tachyons classes instead of applying some of these styles, like `list` and  `overflow-y-auto`. You can check it out all the available classes [here](https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css)

Reply:
I saw that you were just passing these styles from another file, but could you do this?

Reply:
No problem! ; )

Reply:
No problem!